flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/physical/stream/StreamExecSinkRule.scala

```diff
@@ -53,15 +53,20 @@ class StreamExecSinkRule extends ConverterRule(
             val dynamicPartIndices =
               dynamicPartFields.map(partitionSink.getTableSchema.getFieldNames.indexOf(_))
 
-            if (partitionSink.configurePartitionGrouping(false)) {
-              throw new TableException("Partition grouping in stream mode is not supported yet!")
-            }
+            val shuffleEnable = sinkNode
+                .catalogTable
+                .getProperties
+                .get(FileSystemTableFactory.SINK_SHUFFLE_BY_PARTITION.key())
 
-            if (!partitionSink.isInstanceOf[DataStreamTableSink[_]]) {
```

Comment:
`DataStreamTableSink` is a special Type, this change will make all stream sink nodes have chance to use the shuffle enable config?

Reply:
See above `case partitionSink: PartitionableTableSink`, it should never be a `DataStreamTableSink`.

Reply:
`DataStreamTableSink` is not a user interface, it is a internal util class.

Reply:
I got your point, it's redundant here,  because `DataStreamTableSink` only extends `TableSink` from now.